app/script/util/Environment.js

```diff
@@ -47,43 +47,40 @@ window.z.util = z.util || {};
 
   const _check = {
     getVersion: () => (window.platform.version ? window.parseInt(window.platform.version.split('.')[0], 10) : null),
-    isChrome() {
-      return window.platform.name === BROWSER_NAME.CHROME || this.isElectron();
-    },
-    isDesktop() {
-      return this.isElectron() && window.platform.ua.includes(BROWSER_NAME.WIRE);
-    },
+    isChrome: () => window.platform.name === BROWSER_NAME.CHROME || this.isElectron(),
+    isDesktop: () => this.isElectron() && window.platform.ua.includes(BROWSER_NAME.WIRE),
```

Comment:
I did that as well.. now wait for Travis ❌

Reply:
Yeah, makes sense. Cleaned up the whole file and turned it into a proper self invoking function. Should be all good now.